<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计数器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonCntUp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string extracomment="114514">+1</string>
     </property>
     <property name="icon">
      <iconset theme="address-book-new">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCntDn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCnt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCntRst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonCntUp</tabstop>
  <tabstop>pushButtonCntDn</tabstop>
  <tabstop>lineEditCnt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCntUp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntUp_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCntRst</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntRst_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>-10</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCntDn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntDn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>-41</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnCntUp_Click()</slot>
  <slot>btnCntRst_Click()</slot>
  <slot>btnCntDn_Click()</slot>
  <slot>test()</slot>
 </slots>
</ui>
